wdl/meta.wdl

```diff
@@ -315,3 +349,55 @@ task post_filter {
         noAddress: true
     }
 }
+
+# Gather qc metrics to one file
+task gather_qc {
+
+    input {
+        Array[Array[File]] qc
+        String docker
+
+        Array[File] qcs = flatten(qc)
+    }
+
+    command <<<
+
+        Rscript - <<EOF
+
+        suppressPackageStartupMessages(library(data.table))
+
+        files <- strsplit(x = "~{sep=" " qcs}", split = " ")[[1]]
+        files_list <- split(files, gsub(".*[.]tsv[.]gz[.]qc[.]|[.]tsv", "", files))
+
+        merged_list <- lapply(files_list, function(x) {
+            do.call(rbind, lapply(x, fread))
+        })
+
+        for(i in names(merged_list)) {
+            fwrite(x = merged_list[[i]],
+                    file = paste0("qc.", i, ".tsv"),
+                    sep = "\t",
+                    quote = F,
+                    col.names = T,
+                    row.names = F,
+                    na = "NA")
+        }
+
+        EOF
```

Comment:
noice!!! very nice clean and concise!!!  Not putting all into one tube for readability but using nice functional way of making code short!